modules/ossm-kiali-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-kiali.adoc
+////
+
+[id="ossm-kiali-architecture_{context}"]
+= Kiali architecture
+
+Kiali is composed of two components: the Kiali application and the Kiali console.
+
+* *Kiali application* (back end) – This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in configmaps and secrets.
+
+* *Kiali console* (front end) – The Kiali console is a web application. 	The Kiali application serves the Kiali console, which then queries the application for data in order to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+* *Red Hat Service Mesh* (Istio) - Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and won’t work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the cluster API.
+
+* *Prometheus* - A dedicated Prometheus instance is included as part of the {ProductName} installation.  When Istio telemetry is enabled, metrics data is stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, etc. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetery. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali’s features will not work without Prometheus.
+
+* *Cluster API* - Kiali uses the API of the OpenShift container application platform (cluster API) in order to fetch and resolve service mesh configurations. Kiali queries the cluster API to retrieve, for example, definitions for: namespaces, services, deployments, pods, and other entities. Kiali also makes queries to resolve relationships between the different cluster entities. The cluster API is also queried to retrieve Istio configurations like: virtual services, destination rules, route rules, gateways, quotas, etc.
```

Comment:
* Apologies if this is just my ignorance, but is "OpenShift container application platform" actually different than "OpenShift Container Platform"? If not, then you should use the {product-title} attribute for OCP in our document attribtues file.
* I don't think the colon after "for" or "like" is necessary.

Reply:
No, that should be OpenShift Container Platform.